Description: Fixing Makefile installation path
 The original upstream makefile installs the program to /usr/bin, but we want
 it under /usr/games. This has not been forwarded upstream, as it is a
 conscious difference in policy between what upstream wants and what we want.
Author: Felix Crux <felixc@felixcrux.com>
Forwarded: no
Last-Update: 2013-05-13

--- a/Makefile
+++ b/Makefile
@@ -6,7 +6,7 @@
 INSTALL=install -D
 INSTALL_DATA=$(INSTALL) -m 644
 
-BINDIR=$(DESTDIR)/usr/bin
+BINDIR=$(DESTDIR)/usr/games
 MANDIR=$(DESTDIR)/usr/share/man/man6
 
 PROG=gti
